aictl/cli/main.py

```diff
@@ -112,6 +112,36 @@ def ip2p(args):
     print("### saving image files")
     output.images[0].save(args.output_path)
 
+def t2v(args):
+    import torch
+    from diffusers import DiffusionPipeline
+    from diffusers.utils import export_to_video
+
+     # check if on mac and mps is available, fallback to cuda then cpu
+    is_mac = False
+    device = "cuda:0" if torch.cuda.is_available() else "cpu"
+    if torch.backends.mps.is_available():
+        is_mac = True
+        device = torch.device("mps")
+        print("MPS device detected. Using MPS.")
+
+    pipe = DiffusionPipeline.from_pretrained("damo-vilab/text-to-video-ms-1.7b", torch_dtype=torch.float16, variant="fp16")
+
+
+    # memory optimization
+    if is_mac:
+        pipe.enable_attention_slicing()
+    else:
+        pipe.enable_vae_slicing()
+        pipe.enable_model_cpu_offload()
+        pipe.enable_xformers_memory_efficient_attention()
+    pipe = pipe.to(device)
+     
+    print(args.prompt)
+    video_frames = pipe(args.prompt, num_frames=args.frames).frames
+    video_path = export_to_video(video_frames)
+    print(video_path)
```

Comment:
Is there anyway we can add an `--output-path` argument here to allow the user to specify where they want the file and what name they want to use?